tinkerbell/stack/values.yaml

```diff
@@ -21,6 +23,24 @@ stack:
     # downloadURL only works with the > 0.8.1 Hook release because
     # previous Hook versions didn't provide a checksum file.
     downloadURL: https://github.com/tinkerbell/hook/releases/download/latest
+    persistence:
+      enabled: true
+      ## type hostpath or pvc
+      # hostpath : only works on a single worker node cluster
+      type: hostpath
```

Comment:
Sorry to throw a spanner in the works, but when I looked at this again it made me wonder if we'd be better using keys to represent the persistence type avoiding the 'only use this if using pvc' commentary about each of the items below.

E.g.

```yaml
hook:
	persistence:
		hostpath: {}
		pvc:
			storageClassName: ""
			...
```

@jacobweinstock Do you have a preference?

Reply:
With your suggestion we could still use `type: ` and then both `hostpath:` and `pvc:` could be populated without issue.

```yaml
hook:
  persistence:
    # type must be either "hostpath" or "pvc"
    # hostpath: only works on a single worker node cluster
    type: hostpath
    hostpath: {}
    pvc:
      storageClassName: ""
      ...
```

Reply:
I'm open to that, though I don't think its strictly necessary because we can test for the presence of either `hostpath` or `pvc` keys. I'm not sure which is clearer so I'm good either way.